pkg/arduino_adafruit_sensor/Makefile.dep

```diff
@@ -1,4 +1,3 @@
-FEATURES_REQUIRED += arduino
 FEATURES_REQUIRED += cpp
```

Comment:
This is also technically not needed, as arduino depends on `cpp`. However, the package uses C++ code itself, (and not only indirectly via the `arduino` module), so I think we should keep this one.